tests/framework/db/sqlite/QueryBuilderTest.php

```diff
@@ -143,6 +143,37 @@ public function testBuildUnion()
         $this->assertEquals([], $queryParams);
     }
 
+    public function testBuildWith()
```

Comment:
Please rename test methods to name of original methods with prefix `test`.
If testable method named as `buildWithQueries` then name test method `testBuildWithQueries`.

Reply:
Ok. Thank you for review